ui/oopsyraidsy/data/innocence-ex.js

```diff
@@ -31,4 +31,5 @@
     'InnoEx God Ray 3': '3EE8',
     'InnoEx Explosion': '3EF0',
   },
+  triggers: [],
```

Comment:
This shouldn't be needed.  Is this still giving errors on the latest of master branch?

Reply:
You are right, its not longer needed. Reverted commit.